mlos_bench/mlos_bench/optimizers/base_optimizer.py

```diff
@@ -10,6 +10,7 @@
 import logging
 from typing import Dict, Optional, Sequence, Tuple, Union
 from abc import ABCMeta, abstractmethod
+from distutils.util import strtobool    # pylint: disable=deprecated-module
```

Comment:
I see no mention that this particular bit is deprecated in the docs upstream and this is a handy method to use, so I've just silenced the message for now.  We should catch it in our CI pipelines if that does actually get deprecated.

https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool